q2_feature_table/plugin_setup.py

```diff
@@ -652,3 +654,59 @@
                 'splits are defined by values in metadata column.',
     examples={}
 )
+
+plugin.methods.register_function(
+    function=q2_feature_table.tabulate_feature_frequencies,
+    inputs={'table': FeatureTable[Frequency]},
+    parameters={},
+    outputs={'metadata': ImmutableMetadata},
+    input_descriptions={
+        'table': 'A table containing feature frequency information'
+    },
+    output_descriptions={
+        'metadata': 'Metadata generated from frequency data'
+    },
+    name='Tabulate feature frequencies',
+    description='Tabulates feature frequencies from a feature table '
+                'and generates metadata',
+    examples={}
+)
+
+plugin.methods.register_function(
+    function=q2_feature_table.tabulate_sample_frequencies,
+    inputs={'table': FeatureTable[Frequency]},
+    parameters={},
+    outputs={'metadata': ImmutableMetadata},
+    input_descriptions={
+        'table': 'A table containing sample frequency information'
+    },
+    output_descriptions={
+        'metadata': 'Metadata generated from frequency data'
+    },
+    name='Tabulate sample frequencies',
+    description='Tabulates feature frequencies from a feature table '
+                'and generates metadata',
+    examples={}
+)
+
+plugin.pipelines.register_function(
+    function=q2_feature_table.summarize_plus,
+    inputs={'table': FeatureTable[Frequency | RelativeFrequency |
+                                  PresenceAbsence]},
+    parameters={'metadata': Metadata},
+    outputs={'frequency_metadata': ImmutableMetadata,
+             'sample_metadata': ImmutableMetadata,
+             'visualized_data': Visualization},
+    input_descriptions={
+        'table': 'The feature table to be summarized.'
+    },
+    parameter_descriptions={'sample_metadata': 'The sample metadata.'},
+    output_descriptions={'frequency_metadata': 'Metadata pertaining to '
+                         'feature frequencies',
+                         'sample_metadata': 'Metadata pertaining to '
+                         'sample frequencies',
```

Comment:
Standardize the language here with the language used in the individual actions. Right now this says "pertaining to" and the actions say "generated from." I'd probably go with "generated from" for both of them.